Dockerfile

```diff
@@ -22,28 +23,30 @@ RUN mkdir /etc/service/php5-fpm
 ADD runit/php5-fpm.sh /etc/service/php5-fpm/run
 
 ADD config/nginx.conf /etc/nginx/nginx.conf
-ADD config/nginx-default.conf /etc/nginx/sites-available/default
+ADD config/nginx-piwik.conf /etc/nginx/sites-enabled/piwik.conf
+ADD config/nginx-piwik-ssl.conf /etc/nginx/sites-enabled/piwik-ssl.conf
 ADD config/php.ini /etc/php5/fpm/php.ini
 
 RUN cd /usr/share/nginx/html && \
-    export PIWIK_VERSION=2.10.0 && \
-    wget http://builds.piwik.org/piwik-${PIWIK_VERSION}.tar.gz && \
-    tar -xzf piwik-${PIWIK_VERSION}.tar.gz && \
-    rm piwik-${PIWIK_VERSION}.tar.gz && \
+    wget http://builds.piwik.org/piwik.tar.gz && \
```

Comment:
I'll keep the explicit piwik version, but I'll bump it to 2.13.1.  I started out always pulling the "latest" tarball, and got burned by a major database format upgrade, so I moved to explicit version.  I think its probably clearer to people coming along, as well, that stuff in here is intended for 2.13.1, so if they want to try bumping it to 3.7.0, they may have a bad day.  I've also been meaning to really play with upgrades in a running container, so that even if the container lags a version or two, people can always just upgrade it once its running.